tools/fuzzer_corpus.cc

```diff
@@ -122,7 +122,8 @@ struct ImageSpec {
     bool modular_mode = false;
     bool lossy_palette = false;
     bool noise = false;
-    uint8_t padding_[1] = {};
+    bool preview = false;
+    // uint8_t padding_[0] = {};
```

Comment:
maybe add a comment on why is this commented here; like that we need to add padding because "packed".

Reply:
Done.